govcd/edgegateway_test.go

```diff
@@ -5,192 +5,117 @@
 package govcd
 
 import (
-	"github.com/vmware/go-vcloud-director/testutil"
+	"fmt"
 	types "github.com/vmware/go-vcloud-director/types/v56"
 	. "gopkg.in/check.v1"
 )
 
-func (vcd *TestVCD) Test_Refresh(c *C) {
-
-	testServer.ResponseMap(2, testutil.ResponseMap{
-		"/api/vdc/00000000-0000-0000-0000-000000000000/edgeGateways":  testutil.Response{200, nil, edgegatewayqueryresultsExample},
-		"/api/admin/edgeGateway/00000000-0000-0000-0000-000000000000": testutil.Response{200, nil, edgegatewayExample},
-	})
-
-	edge, err := vcd.vdc.FindEdgeGateway("M916272752-5793")
-	_ = testServer.WaitRequests(2)
-	testServer.Flush()
-
-	c.Assert(err, IsNil)
-	c.Assert(edge.EdgeGateway.Name, Equals, "M916272752-5793")
-
-	testServer.Response(200, nil, edgegatewayExample)
+func (vcd *TestVCD) Test_Refresh(check *C) {
+	if vcd.config.VCD.EdgeGateway == "" {
+		check.Skip(fmt.Sprintf("skipping %s because no edgegatway given", check.TestName()))
```

Comment:
`Skip()` provides already the name of the test (when we run the tests with `-check.v`)